apps/omg_watcher/priv/swagger/watcher_api_specs.yaml

```diff
@@ -474,20 +474,20 @@ responses:
                   competing_sig:
                     type: string
                     format: binary
-                  competing_txid:
+                  competing_tx_pos:
                     type: integer
                     format: int256
                   competing_proof:
                     type: string
                     format: binary
             example:
               data:
-                inflight_txbytes: '0xf3170101c0940000...'
-                inflight_input_index: 1
+                in_flight_txbytes: '0xf3170101c0940000...'
+                in_flight_input_index: 1
                 competing_txbytes: '0x5df13a6bee20000...'
                 competing_input_index: 1
                 competing_sig: '0xa3470101c0940000...'
-                competing_txid: 2600003920012
+                competing_tx_pos: 2600003920012
```

Comment:
Following conversation [here](https://github.com/omisego/elixir-omg/pull/403#discussion_r252998294):
```
iex(3)> 2600003920012 |> Utxo.Position.decode()
{:utxo_position, 2600, 392, 12}
```
Above have 2 drawbacks:
1. Suggest impossible transaction in deposit block
1. Misleading 12th oindex

Can you please improve these accordingly? Thanks!